### PR TITLE
Bg 2963 min confirms max height list tx

### PR DESCRIPTION
**Pull request by Xekyo**
Added upper bound to transaction block height in order to fix timeout issue when looking up very old transactions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "argparse": "0.1.16",
     "assert": "0.4.9",
-    "bitgo": "4.17.0",
+    "bitgo": "4.19.0",
```
Comment (Melvillian) [inline package.json]: The latest version of the SDK is 4.18.1 (both in the repo and on NPM), so I'm going to wait on merging this until 4.19.0 has been published.

Comment (Xekyo) [reply package.json]: @Melvillian: Yes, this PR is waiting for 4.19.0 to be published.

**Comment by Melvillian** [comment]
@Xekyo I'll make a new 4.19.0 tag on BitGoD and get it published today

**Comment by Melvillian** [comment]
Nevermind, it's already there!

**Comment by Xekyo** [comment]
Yep, I've also updated it to actually link to the merge commit.

**Pull request merged**